### ISSUE TITLE
Disable Current Notification Daemon so AGS Will Work (KDE Plasma base)

### ISSUE THREAD

**Issue by mrstephenhanson** [Jas-SinghFSU/HyprPanel#117]
Greetings - 

I was able to install HyprPanel and add the autostart to my hyprland.conf. However, AGS is unable to load. I'm looking at this particular section in the instructions: 

"Notifications
HyprPanel handles notifications through the AGS built-in notification service. If you're already using a notification daemon such as Dunst or Mako, you may have to stop them to prevent conflicts with HyprPanel."

I do not have Dunst or Mako installed, and I'm not 100% sure what my notification daemon is. Probably plasma notifications. 

How can I switch from plasma notifications to ags?

Thank you greatly!

**Comment by Jas-SinghFSU** [comment]
Are you getting anything in the logs when you start ags?

**Comment by mrstephenhanson** [comment]
> Are you getting anything in the logs when you start ags?

Woah thank you for the quick response!

I sure am and it used to be several pages long. I just tested again and it's much more manageable, and I can see that my notification daemon is SwayNotificationCenter. 

Here's the output in its entirety: 

"(com.github.Aylur.ags:40434): Gjs-Console-WARNING **: 09:47:24.810: Another notification daemon is already running: SwayNotificationCenter

(com.github.Aylur.ags:40434): Gjs-Console-CRITICAL **: 09:47:24.886: Error: Requiring GnomeBluetooth, version 3.0: Typelib file for namespace 'GnomeBluetooth', version '3.0' not found
require@resource:///org/gnome/gjs/modules/esm/gi.js:16:28
@gi://GnomeBluetooth/?version=3.0:3:25
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:266:34
  

(com.github.Aylur.ags:40434): Gjs-CRITICAL **: 09:47:24.914: Attempting to call back into JSAPI during the sweeping phase of GC. This is most likely caused by not destroying a Clutter actor or Gtk+ widget with ::destroy signals connected, but can also be caused by using the destroy(), dispose(), or remove() vfuncs. Because it would crash the application, it has been blocked and the JS callback not invoked.
The offending signal was access-point-removed on NMDeviceWifi 0x55864c340b00.
== Stack trace for context 0x55864b831960 =="

I went ahead and installed Gnome-Bluetooth-3.0 out of curiosity, and AGS starts now! It's underneath my current Waybar as its own panel. 

So it looks like everything is good on the AGS end and I just have some conflicting configurations going on. I guess I don't really need any more help unless you know how to get AGS to work as a Waybar entry. ^_^

Thank you!!!
